grcs/0020-knowledge-graph.md

```diff
@@ -148,21 +136,21 @@ There are 6 native value types. A princple used for naming the native types is t
 | Point        | bc76a151-fa46-4a9a-9590-c40e1738f452 | For geo locations or other coordinates                    |
 
 
-In addition, properties can have Value Type: Relation, which isn't a native type, but the allows us to define relation types as part of the schema. For each native value type, if a value is provided that is invalid, the triple should be dropped and treated as unset. Null cannot be provided as a value. Each native value type is UTF-8 encoded as a string.
+In addition, properties can have Value Type: Relation, which isn't a native type, but the allows us to define relation types as part of the schema. 
+
+| Name         | Type      | ID                     |
+| ------------ |-----------|----------------------- |
+| Relation     | Type      | 1dc83461-cb6c-4af7-bca1-84cb9554c84a |
+
+For each native value type, if a value is provided that is invalid, the triple should be dropped and treated as unset. Null cannot be provided as a value. Each native value type is UTF-8 encoded as a string.
```

Comment:
@yanivtal should it be: `the property should be dropped` instead of `the triple should be dropped`?